schema.json

```diff
@@ -29,7 +29,12 @@
     },
     "type": "object",
     "properties": {
+        "$id": {
+            "type": "string",
+            "format": "uriref"
+        },
         "id": {
+            "description": "This keyword has been deprecated in favor of \"$id\"",
```

Comment:
That's probably a good idea, though the patch does include a comment saying it'll likely get removed in the future.

And of course, just because it's removed doesn't mean that implementations can't still support it (i.e. silently upgrade it).

Lemme think about this some, unless you can elaborate more